ambari-server/src/main/resources/Ambari-DDL-Postgres-CREATE.sql

```diff
@@ -159,9 +159,10 @@ CREATE TABLE clusterservices (
   cluster_id BIGINT NOT NULL,
   service_group_id BIGINT NOT NULL,
   service_enabled INTEGER NOT NULL,
-  CONSTRAINT PK_clusterservices PRIMARY KEY (id, service_group_id, cluster_id),
-  CONSTRAINT UQ_service_id UNIQUE (id),
-  CONSTRAINT FK_clusterservices_cluster_id FOREIGN KEY (service_group_id, cluster_id) REFERENCES servicegroups (id, cluster_id));
+  CONSTRAINT PK_clusterservices PRIMARY KEY (id),
+  CONSTRAINT UK_clusterservices_id UNIQUE (id, cluster_id, service_group_id, service_name),
```

Comment:
Since `id` is the primary key, this is guaranteed to be unique even with duplicate values for the rest of the columns.  Shouldn't it constrain `(cluster_id, service_group_id, service_name)` instead?

Reply:
Fixed.